automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/billing/BillingManagementPage.scala

```diff
@@ -52,7 +55,7 @@ class BillingManagementPage(implicit webDriver: WebDriver) extends BaseFireCloud
     */
   def waitForCreateDone(projectName: String): Option[String] = {
     billingProjectTable.filter(projectName)
-    retry(10.seconds, 10.minutes)({
+    retry(10.seconds, 20.minutes)({
```

Comment:
waiting for new billing project creation timed out after 10 min that failed test `BillingSpec`. somedays Google is very sloow.
https://fc-jenkins.dsp-techops.broadinstitute.org/job/firecloud-ui-fiab-test-runner/2990/testReport/junit/org.broadinstitute.dsde.firecloud.test.billing/BillingSpec/A_user_with_a_billing_account_with_a_new_billing_project_should_be_able_add_a_new_user__create_a_workspace__and_run_a_method__change_billing_account/